e2e/cases/resolve/false/index.ts

```diff
@@ -0,0 +1,3 @@
+import jjj from './browser-false/util';
+
+console.log('jjj:', jjj); // the value of `jjj` should `{}`
```

Comment:
`jjj` looks weird, we could change to `value` or `foo` to fix the bad pattern from modern module.